mesa/experimental/jupyter_viz.py

```diff
@@ -51,6 +51,23 @@ def make_model():
         set_current_step(0)
         return model
 
+    def calculate_space_size():
+        if model_parameters["N"] > 50:
+            fixed_size = int(model_parameters["N"] / 5)
+            model_parameters.update({"width": fixed_size, "height": fixed_size})
+            plt.rcParams["figure.figsize"] = (fixed_size, fixed_size)
+
+    def calculate_agent_size():
+        aspect_ratio_high = (
```

Comment:
Definition of aspect ratio is width/height: https://en.wikipedia.org/wiki/Aspect_ratio_(image). `fill_fraction` is a more descriptive term.

Reply:
sure, makes more sense, thanks!